_includes/header.html

```diff
@@ -1,5 +1,20 @@
 
 <!-- Intro Start -->
+<style>
+#icon{
+  position: absolute;
+	bottom:25px;
```

Comment:
is this fine in mobile view too?

Reply:
yeah
![mobile_view_scroll_down](https://user-images.githubusercontent.com/47534164/84808514-651f0900-b026-11ea-9b98-1b798b166244.png)

Reply:
That's great
pull latest changes
then push again

Reply:
`git pull origin master` 
then push